sdk/data_kusto/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+#[macro_use]
+extern crate serde_derive;
```

Comment:
Can we get rid of this and just refer to `Serialize` and `Deserialize` through full paths: e.g., `serde::Deserialize`

Reply:
Done, much cleaner now. Do I understand correctly that the use of  `macro_use` could be considered deprecated in recent Rust versions in favour of direct imports?

Reply:
Deprecated is perhaps too strong of a word, but it is becoming less and less idiomatic over time, and there is movement in the language team to deprecate this perhaps in the next edition.